src/modules/auth/googleauth/google.service.ts

```diff
@@ -0,0 +1,34 @@
+import { PassportStrategy } from '@nestjs/passport';
+import { Strategy, VerifyCallback } from 'passport-google-oauth20';
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class GoogleStrategy extends PassportStrategy(Strategy, 'google') {
+  constructor() {
+    super({
+      clientID: `${process.env.GOOGLE_CLIENT_ID}`,
```

Comment:
if you add something to env, you need to have env.example file in other case rest of the team will have problem with project start

Reply:
fixed